src/shared/components/oncoprint/styles.scss

```diff
@@ -1,9 +1,14 @@
 .oncoprintjs__tooltip {
   padding: 5px;
   margin-top: -10px; // to compensate position for padding, since tooltip is positioned above (if positioned at center, wouldnt be problem)
-  width:250px;
+  max-width: none;
+}
+
+.oncoprint__tooltip {
```

Comment:
?

Reply:
I did not create these classes, but just updated the css to make it work. Style changes to both classes are required for the tooltip.

https://github.com/cBioPortal/cbioportal-frontend/pull/2633#discussion_r317689299
> @alisman i think max-width existed for reason. probably in some scenario there's a large body of text that we want to have wrap (@adamabeshouse ?). to fix this situation more surgically, we could apply word-break: break-all; to the the a tag that is at issue.
> @adamabeshouse thats correct

Ok, I will update the code to use word wrap.

Reply:
See updated Images in the PR description for the tooltips with word-wrap.